packages/shared/lib/core/api/interfaces/api.interface.ts

```diff
@@ -13,7 +13,6 @@ import {
 } from '@iota/sdk/out/types'
 import { IWallet } from '@core/profile/interfaces'
 
-// TODO(2.0): Every method should return a promise (maybe except Utils, needs research)
 export interface IApi {
```

Comment:
It's weird that `computeFoundryId` is a promise and `computeAccountId` and `computeNftId` are not. Is this correct?
On the other hand, doesn't `clearWalletsFromMemory` need a promise?